.changeset/lemon-lamps-shave.md

```diff
@@ -0,0 +1,7 @@
+---
+'@lit/reactive-element': patch
+'lit-element': patch
+'lit': patch
+---
+
+Allow null to be in the type of @query() decorated fields
```

Comment:
Nit: the `@` prefixed names can tag random users when the changelog are on github unless written as code.
```suggestion
Allow `null` to be in the type of `@query()` decorated fields
```